apps/prairielearn/assets/scripts/lib/parallelHistograms.ts

```diff
@@ -0,0 +1,201 @@
+import { scaleLinear, scaleBand, axisBottom, axisLeft, range, select } from 'd3';
+
+interface Data {
+  label: string;
+  histogram: number[];
+  mean: number;
+}
+
+export function parallel_histograms(
```

Comment:
Nit: `parallelHistograms` to match our conventions?

Reply:
Updated to `parallelHistograms` in [4dad954](https://github.com/PrairieLearn/PrairieLearn/pull/9809/commits/4dad954294139e1ccb0018548e102663ab43c2df).